submissions/career/career.md

```diff
@@ -0,0 +1,48 @@
+
+Unfortunately, it wasn't all sunshine and roses in my journey into the depths of Factorio. As is not uncommon, I gleefully enter the game, made and automated the red and green packs then burned out. Looking back, I suspect there was some Complexity Bias going on here, which again is very common in Factorio if you aren't prepared to break down problems and understand their individual components. The game sat by the wayside, I tried other games and left poor Factorio out in the cold, not a friend in the world.
+
+Several months later, my cousin came to visit and saw I had Factorio - he was excited to play it so we decided that together we would give it a go. After some sketchy library sharing (yay Steam offline mode), we spent a weekend of bliss together, figuring out how the game worked and progressing past the ever challenging blue science packs. *Side note: This is a really great example of Piracy for good since that weekend of Factorio caused my cousin to go home and buy the game himself so we could keep playing together, something that might not have happened if not for our dodgy dealings.* This push past the percieved complexity forever ruined me, it caused the crippling Cracktorio addiction that many of us share, meaning I never could put this beast back down.
+
+So... why am I telling you this? *Fantastic question, really glad you asked. Very well asked question I might add.* For me its pretty clear than playing modded Minecraft and Factorio pushed forward my critical thinking and mathmatical skills and diverted me down the STEM path of goodness. Now, I'm studying Robotic Engineering and though Factorio isn't the only thing that pushed me down this path, it definitely help guide me to where I am now.
+
+### The computer science angle
+
+I'm not alone in the experience of Factorio guiding my choices elsewhere in life. So many people I've talked to, including our very own *Therenas - ~~LORD GOD~~ Editor-in-chief of Alt-F4*, have starting playing Factorio, experimented with mods then been forced into learning to program so they can fufill their desire to create their own mods. I would *blindly, and with no evidence* suggest there are a not insignificant number of people who learned to program because of Factorio and other moddable games. Even just playing around with combinators and learning the basic of computer logic provides a new skill for many people, maybe pushing them slightly closer to the ever loved STEM fields.
+
+![Playing to modding Factorio meme](https://cdn.discordapp.com/attachments/603392474458882065/745728165116248144/mod_brain.png)
+*Source: Find creator*
```

Comment:
Hi :) https://discord.com/channels/139677590393716737/306402592265732098/672169819696791582

Reply:
To be clear, I can't 100% remember if I originally made this image, but I think I did. I can't find the original file anymore, the png I have saved is from feb 2020 (after the linked discord message). I don't really care to be credited anyway.